data/hazards.txt

```diff
@@ -124,3 +124,20 @@ hazard "Rulei Gravity Field"
 
 effect "gravity well"
 	"sound" "gravity well"
+
+
+hazard "Core Sheltered Heat"
+	"duration" 1
+	"strength" 0.65 0.85
+	"system-wide"
+	weapon
+		"burn damage" 0.06
+		"relative heat damage" 0.0012
```

Comment:
I explained why, as part of an overarching process which is not yet concluded, this was not changing at this time.
If you choose not to accept that explanation, that's on you. I'm not redoing the math on my side of things for balancing the Xapleaux PR just so it can temporarily look pretty here.